tests/robot/suites/trafficIPv6/ip_route/test1_ip_routeIPv6.robot

```diff
@@ -116,10 +116,10 @@ Pinging
     Ping6 From agent_vpp_1 To ${IP_4}
     #Ping From agent_vpp_2 To ${IP_4}
     ${int}=    vpp_ctl: Get Interface Internal Name    agent_vpp_2    bvi_loop0
-    Ping6 On agent_vpp_2 With IP ${IP_4}, Source ${int}
+    Ping On agent_vpp_2 With IP ${IP_4}, Source ${int}
```

Comment:
Why Ping instead of Ping6 ?

Reply:
Ping for IPv4 and IPv6 is in this case same.